src/owncloud/apache_config

```diff
@@ -14,3 +14,19 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+
+
+# MPM Event
+<If "${SNAP_ARCH} == 'armhf'"">
```

Comment:
@oparoz where did these numbers come from? I assumed they were targeted at the rpi, but this snap runs on more machines than the rpi. Are you saying these numbers would be good for a real amd64 server as well?